packages/flutter_fuchsia/pubspec.yaml

```diff
@@ -0,0 +1,11 @@
+name: flutter_fuchsia
```

Comment:
You will need to run `flutter update-packages --force-upgrade`, which will populate transitive dependencies of this package and make sure the dependencies' version constraints are consistent with the rest of the repo.

Reply:
I believe this is fixed now.  We'll see if things run correctly in Travis, I guess.